src/menus/DisassemblyContextMenu.cpp:527:35: error: attempt to use a deleted function  ->setData(ty->path);

**Environment information**
* Operating System: MacOS 12.6.1 ARM64
* Cutter version: https://github.com/rizinorg/cutter/commit/d47eb1c41fed647e453f81aa6e38961440a61084
* Obtained from:
  - [x] Built from source
  - [ ] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format: -

**Describe the bug**

```
[ 30%] Building CXX object src/CMakeFiles/Cutter.dir/menus/DisassemblyContextMenu.cpp.o
/Users/user/rizin/cutter/src/menus/DisassemblyContextMenu.cpp:527:35: error: attempt to use a deleted function
                        ->setData(ty->path);
                                  ^
/opt/homebrew/lib/QtCore.framework/Headers/qvariant.h:199:5: note: 'QVariant<char *, false>' has been explicitly marked deleted here
    QVariant(T) = delete;
    ^
1 error generated.
```

<!-- A clear and concise description of what the bug is. -->

**To Reproduce**

Steps to reproduce the behavior:
1. Clone Cutter from `dev`
2. `cmake -B build -DCUTTER_QT6=ON -DCUTTER_ENABLE_SIGDB=ON`
3. `make -C build`

**Expected behavior**

No error

--- a/src/menus/DisassemblyContextMenu.cpp
+++ b/src/menus/DisassemblyContextMenu.cpp
@@ -524,7 +524,7 @@ void DisassemblyContextMenu::aboutToShowSlot()
                     continue;
                 }
                 structureOffsetMenu->addAction("[" + memBaseReg + " + " + ty->path + "]")
-                        ->setData(ty->path);
+                        ->setData(QString(ty->path));
             }
             rz_list_free(typeoffs);
         }

